xenforo/library/bdApi/Model/Subscription.php

```diff
@@ -126,11 +126,12 @@ public function ping(array $option, $action, $objectType, $objectData)
         if (!isset($option['topic'])
             || empty($option['subscriptions'])
         ) {
-            return false;
+            return 0;
```

Comment:
```suggestion
            return false;
```

Nếu input thiếu data chắc vẫn để false đi cho rõ?